lua/nvim-autopairs/completion/cmp.lua

```diff
@@ -25,13 +32,18 @@ M.setup = function(opt)
     if map_complete then
         local method_kind = cmp.lsp.CompletionItemKind.Method
         local function_kind = cmp.lsp.CompletionItemKind.Function
+        local filetype = vim.fn.expand("%:e")
+        if map_char[filetype] == nil then
```

Comment:
please reduce using the if condition by the 'or' syntax

local char = mapchar[vim.bo.filetype] or '('